src/tdastro/astro_utils/passbands.py

```diff
@@ -0,0 +1,258 @@
+
+        Note
+        ----
+        This is a temporary solution to be replaced when we implement Pooch throughout the project.
+        """
+        data_dir = os.path.join(Path(__file__).parent, "band_data")
+        if not os.path.exists(data_dir):
+            os.makedirs(data_dir)
+        self.data_path = data_dir
+
+    def _download_transmission_table(self, band_name: str, output_file: str, timeout: int = 5) -> bool:
+        """Download a transmission table for a single band and save it to a file.
+
+        Files used are from:
+        http://svo2.cab.inta-csic.es/svo/theory/fps3/index.php?mode=browse&gname=LSST
+
+        Note
+        ----
+        This is a temporary solution to be replaced when we implement Pooch throughout the project.
+
+        Parameters
+        ----------
+        band_name : str
+            Name of the band for which the transmission table is to be downloaded.
+        output_file : str
+            Path to the file where the transmission table will be saved.
+        timeout : int
+            Time in seconds to wait for the file to be downloaded and created.
+
+        Returns
+        -------
+        bool
+            True if the transmission table is successfully downloaded, False otherwise.
+        """
+        try:
+            socket.setdefaulttimeout(timeout)
+
+            url = (
+                f"http://svo2.cab.inta-csic.es/svo/theory/fps3/getdata.php"
+                f"?format=ascii&id=LSST/LSST.{band_name}"
+            )
+
+            urllib.request.urlretrieve(url, output_file)
+
+            if os.path.getsize(output_file) == 0:
+                logging.error(f"Transmission table for {band_name} is empty.")
+                return False
+
+            logging.info(f"Downloaded: {band_name}-band transmission table.")
+            return True
+        except HTTPError as e:
+            logging.error(f"HTTP error occurred when downloading transmission table for {band_name}: {e}")
+            return False
+        except URLError as e:
+            logging.error(f"URL error occurred when downloading transmission table for {band_name}: {e}")
+            return False
+
+    def _load_transmission_table(self, band_name: str, file_path: str) -> bool:
+        """Load a transmission table from a file.
+
+        Note
+        ----
+        This is a temporary solution to be replaced when we implement Pooch throughout the project.
+
+        Parameters
+        ----------
+        band_name : str
+            Name of the band for which the transmission table is to be loaded.
+        file_path : str
+            Path to the file containing the transmission table.
+
+        Returns
+        -------
+        bool
+            True if the transmission table is successfully loaded, False otherwise.
+        """
+        try:
+            self.transmission_tables[band_name] = np.loadtxt(file_path)
```

Comment:
A consideration for a future PR. I think you might be able to simplify the user's workflow a bit if you always normalize the table as soon as it is loaded (instead of having the user manually call `calculate_normalized_system_response_tables()`.

Reply:
Linking to #61